.github/linters/.python-lint

```diff
@@ -65,7 +65,120 @@ confidence=
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
 disable=print-statement,
-      apply-builtin,
+	zip-builtin-not-iterating,
+	xreadlines-attribute,
+	xrange-builtin,
+	wildcard-import,
+	using-cmp-argument,
+	useless-suppression,
+	useless-object-inheritance,
+	use-symbolic-message-instead,
+	unused-argument,
+	unpacking-in-except,
+	unicode-builtin,
+	unichr-builtin,
+	too-many-statements,
+	too-many-return-statements,
+	too-many-public-methods,
+	too-many-locals,
+	too-many-lines,
+	too-many-branches,
+	too-many-arguments,
+	too-many-ancestors,
+	too-few-public-methods,
+	sys-max-int,
+	suppressed-message,
+	standarderror-builtin,
+	setslice-method,
+	round-builtin,
+	reload-builtin,
+	reduce-builtin,
+	redefined-outer-name,
+	rdiv-method,
+	raw-checker-failed,
+	raw_input-builtin,
+	range-builtin-not-iterating,
+	raising-string,
+	protected-access,
+	parameter-unpacking,
+	old-raise-syntax,
+	old-octal-literal,
+	old-ne-operator,
+	old-division,
+	oct-method,
+	nonzero-method,
+	non-ascii-bytes-literal,
+	no-self-use,
+	no-name-in-module,
+	no-member,
+	no-absolute-import,
+	next-method-defined,
+	next-method-called,
+	missing-module-docstring,
+	missing-function-docstring,
+	missing-class-docstring,
+	metaclass-assignment,
+	map-builtin-not-iterating,
+	long-suffix,
+	long-builtin,
+	logging-fstring-interpolation,
+	locally-disabled,
+	line-too-long,
+	invalid-str-codec,
+	invalid-name,
+	intern-builtin,
+	input-builtin,
+	indexing-exception,
+	import-star-module-level,
+	import-self,
+	import-outside-toplevel,
+	import-error,
+	idiv-method,
+	hex-method,
+	global-statement,
+	getslice-method,
+	fixme,
+	filter-builtin-not-iterating,
+	file-ignored,
+	file-builtin,
+	execfile-builtin,
+	exception-message-attribute,
+	exception-escape,
+	eval-used,
+	eq-without-hash,
+	duplicate-code,
+	div-method,
+	dict-view-method,
+	dict-values-not-iterating,
+	dict-keys-not-iterating,
+	dict-iter-method,
+	dict-items-not-iterating,
+	deprecated-urllib-function,
+	deprecated-types-field,
+	deprecated-sys-function,
+	deprecated-string-function,
+	deprecated-str-translate-call,
+	deprecated-pragma,
+	deprecated-operator-function,
+	deprecated-itertools-function,
+	delslice-method,
+	dangerous-default-value,
+	cyclic-import,
+	comprehension-escape,
+	coerce-method,
+	coerce-builtin,
+	cmp-method,
+	cmp-builtin,
+	buffer-builtin,
+	broad-except,
+	basestring-builtin,
+	bare-except,
+	bad-python3-import,
+	bad-inline-option,
+	backtick,
+	attribute-defined-outside-init,
+	arguments-differ,
+	apply-builtin,
    	too-many-instance-attributes,
 	useless-with-lock,
```

Comment:
This does not look like a complete sort, right? I see "z .... a", and then "too-many"... Auto, why did you use descenting order?

Reply:
I thought that was the right way to sort the warnings. I'll use ascending order this time.